### PR TITLE
changed: group set related info in a struct

### PR DESCRIPTION
**Pull request by notspiff**
just a small cleanup.

### CODE REVIEW

File: xbmc/video/VideoInfoTag.cpp [hunk 1]
```diff
@@ -221,12 +221,12 @@ bool CVideoInfoTag::Save(TiXmlNode *node, const std::string &tag, bool savePathI
...
-  if (!m_strSet.empty())
+  if (!m_set.title.empty())
   {
     TiXmlElement set("set");
-    XMLUtils::SetString(&set, "name", m_strSet);
-    if (m_strSetOverview.empty())
-      XMLUtils::SetString(&set, "overview", m_strSetOverview);
+    XMLUtils::SetString(&set, "name", m_set.title);
+    if (m_set.overview.empty())
+XMLUtils::SetString(&set, "overview", m_set.overview);
```
Comment (Rechi) [inline xbmc/video/VideoInfoTag.cpp]: indentation is wrong

Comment (notspiff) [reply xbmc/video/VideoInfoTag.cpp]: indeed. will wait for build to finish before pushing fix.

**Comment by notspiff** [comment]
build was all green.

**Comment by Rechi** [comment]
@notspiff ready for merge?

**Pull request merged**